integration/kubernetes/k8s-custom-dns.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="custom-dns-test"
+	file="/etc/resolv.conf"
```

Comment:
`file`  is also a command, you might want to call it something else just for clarity :-)